apps/reshard/provision/src/cli.rs

```diff
@@ -0,0 +1,51 @@
+//! CLI for reshard provisioning.
+
+use clap::Parser;
+use std::path::PathBuf;
+
+use crate::{run, Config};
+
+#[derive(Parser, Debug)]
+#[command(
+    name = "reshard_provision",
+    version,
+    about = "Offline Yubikey provisioning ceremony orchestrator"
+)]
+struct Args {
+    /// Number of operators
+    #[arg(long)]
+    num_operators: usize,
+
+    /// Keys per operator (default: 3)
```

Comment:
```suggestion
    /// Keys per operator
```
It would automatically show the default when you do `--help`